### ISSUE TITLE
Tag new releases

### ISSUE THREAD

**Issue by ezekg** [humanmade/Mercator#37]
It's rather risky for us to be tied to using a master branch on a production site. Anyway that you guys could start tagging your releases, so that we can use version numbers in our `composer.json`? :+1:

**Comment by rmccue** [comment]
Makes sense; we need to sort out a release model, but I'll try and get a 1.0 out this week :)

**Comment by ezekg** [comment]
Please do. When using internal libraries, this is our basic Git workflow for releasing a new version:

```bash
git add -A
git commit -m 'add some cool new feature'
git tag 1.1.0 # Or whatever your new version is
git push origin master --tags
```

Works well and keeps everything tagged nicely.

**Comment by adambiggs** [comment]
+1

**Comment by spacedmonkey** [comment]
+1, if I am going to use it in production, it needs version, so I can easily rollback to fixed version number.

**Comment by ezekg** [comment]
@rmccue can you make this a priority? If not, is it cool if my team forks this repo so that we can keep updates versioned nicely? We're relying on this for a few production sites, and can't afford to have any  breaking changes being introduced into master. We're pulling this in with Composer, so we're not just keeping a local copy.

**Comment by spacedmonkey** [comment]
I think that any code that is as important as this should be tagged with versions. It should also have a change log. There is no way that we could use this in production without the ability to lock down on a version number, if a newer version was buggy. 

cc: @willmot

**Comment by rmccue** [comment]
Just as a heads up, you should be able to use it by specific commits until we get a release out the door:

```json
{
    "require": {
        "humanmade/mercator": "dev-master#80004af9260241958da9024a4cbd63f31fd38ad4"
    }
}
```

We'll get to releasing this proper, just a hectic time at HM right now as we push a bunch of releases out. As part of that, we've been working on Mercator, hence the recent changes. It'll all settle down in a few weeks, and we'll get Mercator all sorted and packaged up for you. :)

**Comment by ezekg** [comment]
:+1:

**Comment by vinkla** [comment]
@rmccue any chance you can tag a release?

**Comment by craigmillerdev** [comment]
@rmccue Please can you start to tag your releases? Your suggested workaround (locking to a sha) is a documented liability for use on a production site. https://getcomposer.org/doc/04-schema.md#package-links. Tagging takes just a few seconds and you'll likely save a lot of headaches by doing it.

Thanks!

**Comment by roborourke** [comment]
@craigmillerdev @vinkla @ezekg @spacedmonkey @adambiggs I'm going to be doing most of the Mercator maintenance in future, as soon I've fixed #68 I'll be tagging version 1 and updating to version 2 which separates the concerns of this plugin. SSO and the admin UI will be split out and additional features made available as part of a wider Mercator ecosystem such as the REST API endpoint.

**Comment by roborourke** [comment]
@craigmillerdev @vinkla @ezekg @spacedmonkey @adambiggs ok, version 1.0.0 tagged here:

https://github.com/humanmade/Mercator/releases/tag/1.0.0

Thanks for bearing with us. I'll be switching master to the v2 ecosystem soon when I have the network admin portion of the UI plugin updated to use the REST API & backbone.
